lib/commands/changelog.js

```diff
@@ -11,9 +13,23 @@ module.exports = {
 
   availableOptions: [],
 
-  anonymousOptions: [],
+  anonymousOptions: [
+    '<from-rev>',
+    '<release-name>',
+  ],
```

Comment:
Yeah, that's what I want to do, see [this branch](https://github.com/lytics/ember-cli-release/commit/62a833a258eb55c559695bdfced24e23cabf164c) over on `ember-cli-release`.

The command args are separate from how release will use it, but are necessary to include if the task is to be self-contained, i.e. if these weren't provided on the command line, the config would have to be read in before the task is invoked.

This is part of a larger question I had about how you see the `changelog` command being used by itself. The way I understand it, there's two options:

1. Suggest that `ember changelog` be set as the `version` hook of [`npm version`](https://docs.npmjs.com/cli/version), use the updated version as the new release/tag name and `head` option, leaving the user to specify what commit to start from.
2. Assume changelog updates happen before version bump/tagging and use the package version as the start commit, leaving the user to specify the release/tag name.

Either way I think it requires the user to specify more information to the command. Hope I'm not missing something obvious.